app/styles/partials/nav-bar.scss

```diff
@@ -0,0 +1,10 @@
+.right.menu > .item > .ui.blue.button {
+  text-transform: uppercase;
+  font-weight: 400;
+}
+
+@media screen and (max-width: 768px) {
+  .right.menu > .item > .ui.blue.button {
+    button-text: small text;
```

Comment:
it is previously present in the anchor tag and I blindly put this in this file but I checked it now it doesn't do anything so sorry, removing it.